app/src/main/java/org/simple/clinic/registration/phone/loggedout/LoggedOutOfDeviceEffectHandler.kt

```diff
@@ -14,8 +15,28 @@ class LoggedOutOfDeviceEffectHandler @Inject constructor(
   fun build(): ObservableTransformer<LoggedOutOfDeviceEffect, LoggedOutOfDeviceEvent> = RxMobius
       .subtypeEffectHandler<LoggedOutOfDeviceEffect, LoggedOutOfDeviceEvent>()
       .addTransformer(LogoutUser::class.java, logoutUser())
+      .addTransformer(ThrowError::class.java, throwError())
```

Comment:
Instead of adding a transformer, can we use a consumer instead?

```kotlin
.addConsumer(ThrowError::class.java, { throw it.cause }, schedulers.ui())
```